src/utility.js

```diff
@@ -92,7 +92,18 @@ function sum(x) {
   return x.reduce((a, b) => a + b, 0);
 }
 
-function mergeInto(obj, other) {
+// if allowOverride is false, it shows error in case of symbol redefinition
+function mergeInto(obj, other, allowOverride = true) {
+  // check for unintended symbol redefinition
+  if (!allowOverride) {
+    for (const key of Object.keys(other)) {
+      if (obj.hasOwnProperty(key)) {
+        error('Symbol redefinition: ' + key);
```

Comment:
Can you advise that the allowOverride option exists?  

Something like `\`Symbol re-definition in JavaScript library: ${key}.  Use allowOverride if this is intended\``.